packages/ui-components/src/InlineAlert/InlineAlert.test.tsx

```diff
@@ -1,51 +1,26 @@
 import React from "react";
-import { shallow, mount } from "enzyme";
-import { IconIntent } from "../Icon/Icon";
+import { render, screen } from "@testing-library/react";
 import { InlineAlert } from "./InlineAlert";
 
-const alertIntents: IconIntent[] = [
-  undefined, // should be used default intent
-  "warning",
-  "success",
-  "danger",
-  "info",
-];
-
 describe("InlineAlert", () => {
-  alertIntents.forEach((intent) => {
```

Comment:
I just spend about an hour trying to figure out a way to test _intents_ or colors with React Testing Library and came up short (because Jest won't use babel to compile library code and I can't find a format for `style-dictionary` that will satisfy both Jest and TypeScript, I push up an example).

Do we currently have a strategy to test that styles are being applied correctly in RTL?

Reply:
Here was my attempt. https://github.com/cockroachdb/ui/pull/430

Reply:
RTL testing philosophy focuses on behavior, so if a style doesn't change behavior, it's going to be difficult to test using RTL helpers.

It's still possible to check that classes are being applied using Jest helpers and Javascript, but testing that classes are being applied doesn't give us confidence that everything looks correct. For instance, a class could be correctly added to an element, but the class in the stylesheet has a typo so no style is applied. The test would pass, but the component would have a bug.

Visual snapshot testing is the best tool to ensure there are no style bugs.

Reply:
Agreed. I can see the difference between having a component behaving in a certain way based on props and "looking correct" (which should involve taking a visual snapshot).